metadata/site_import/meta/system-objecttype-extensions.xml

```diff
@@ -386,6 +386,24 @@
                 <externally-managed-flag>false</externally-managed-flag>
                 <min-length>0</min-length>
             </attribute-definition>
+            <attribute-definition attribute-id="Adyen_Frontend_Region">
+                <display-name xml:lang="x-default">Checkout frontend region</display-name>
```

Comment:
Do we have to specify it is "frontend"? Or can we just call it "Checkout region" instead?

Reply:
I'm half assuming some merchants won't know our cartridge uses Web Components